qml/keys/ActionKey.qml

```diff
@@ -52,9 +52,10 @@ CharKey {
         id: iconImage
         source: Qt.resolvedUrl("../images/" + formFactor + "/" + __icon + ".png")
         anchors.centerIn: parent
+		anchors.verticalCenterOffset: Units.gu(-0.25)
         visible: (label == "")
-       // width: units.gu(2.5)
-       // height: units.gu(2.5)
+        // width: units.gu(2.5)
+        height: parent.height * 0.7
```

Comment:
This makes sure that we don't really need the icons to be properly sized, so we can use the high resolution ones for example.